src/index.tsx

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
-import { Route, Switch } from 'react-router-dom';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
 import MainPage from './views/MainPage';
 
 export default function App() {
   return (
-    <Switch>
-      <Route path="/training" component={MainPage} />
-    </Switch>
+    <BrowserRouter>
```

Comment:
Stripes should be providing a router in the context already; if you need a `<BrowserRouter>`, add it in your test